lib/Common/Exceptions/ExceptionCheck.h

```diff
@@ -94,17 +94,17 @@ class AutoFilterExceptionRegion
 #define AUTO_NO_EXCEPTION_REGION
 #endif
 
-#if DBG
-class DebugCheckNoException
+#ifdef EXCEPTION_CHECK
+class CheckNoException
```

Comment:
Should this `#ifdef` block be merged with the previous one since they now both check for `EXCEPTION_CHECK`?